### PR TITLE
fix(workspace): correct RT addresses in Trading Workspace

### PR DESCRIPTION
**Pull request by algreasley**

### CODE REVIEW

File: src/workspace/config/snapshot.json [hunk 1]
```diff
@@ -327,7 +327,7 @@
                                 },
                                 "webAPIs": ["notifications", "audio", "video"]
                               },
-                              "url": "<RT_URL>/tiles",
+                              "url": "<RT_URL>/fx-fx-tiles",
```
Comment (SHolleworth) [inline src/workspace/config/snapshot.json:330]: Looks like a mistake `fx-fx`

Comment (algreasley) [reply src/workspace/config/snapshot.json:330]: yes, damn!

Comment (algreasley) [reply src/workspace/config/snapshot.json:330]: weird that it worked (I did test it !!) .. that section must not be active

Comment (algreasley) [reply src/workspace/config/snapshot.json:330]: fixed now

**Review by SHolleworth — APPROVED** [review]

**Pull request merged**